README.md

```diff
@@ -36,12 +36,96 @@ TODO
 
 Prometheus is an open-source systems monitoring and alerting toolkit: https://prometheus.io/
 
-#### Add option to log slow queries and recording rules
-* Description: Having something like PostgreSQL's log_min_duration_statement would be useful to debug performance problems. It would be great to collect detailed query information, like how many chunks were necessary to compute the result and how many had to be loaded from disk.
+Prometheus ideas:
+
+#### Streaming APIs
+* Description: Currently the HTTP API sends all the data in one go and that has the chance overwhelm prometheus. To guard against this, there are limits set in Prometheus. Make APIs streaming where data is retrieved and sent in chunks rather than at once.
```

Comment:
I'd be interested in this wrt the remote APIs - the remote read API really needs extending to be streaming, and I'd love to make it use gRPC client-side push.  Do you think this is related enough to be part of the same project, or a different project?

Reply:
I think this is sort of an umbrella project for those efforts, and remote APIs are a part of that, but could be implemented incrementally.